src/components/User.js

```diff
@@ -1,8 +1,18 @@
 import React, { Component } from "react";
-import { Row } from "react-bootstrap";
+import { Row, Col } from "react-bootstrap";
 import Axios from "axios";
 import PullRequest from "./PullRequest";
 import Load from "./Load";
+
+const style = {
```

Comment:
Can you use different file for css?

Reply:
Where would be a good way to place them? Just in the components folder? Or we separate them into a styles folder?

Reply:
I forgot that there is a assets folder. I'll put them there.